.github/workflows/linux-tests.yml

```
@@ -37,7 +37,9 @@ jobs:
            python-version: ${{ matrix.PYTHON_VERSION }}
       - name: Install dependencies
         shell: bash -l {0}
-        run: conda env update --file requirements/environment.yml
+        run: |
+          conda env update --file requirements/environment.yml
+          pip install https://github.com/jupyter/jupyter_client/archive/master.zip --force-reinstall
```

Per this conversation (https://github.com/jupyter/qtconsole/pull/493#discussion_r683777815) the recommended approach would be the following across the three platforms:
```suggestion
          conda install entrypoints nest-asyncio
          pip install https://github.com/jupyter/jupyter_client/archive/master.zip --no-deps
```